sdk/lib/cmlib/cmcheck.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * PROJECT:         ReactOS Registry Manipulation Library
+ * LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:         Registry Validation
+ * COPYRIGHT:       Copyright 2022 George Bișoc <george.bisoc@reactos.org>
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include "cmlib.h"
+#define NDEBUG
+#include <debug.h>
+
+/* DEFINES  ******************************************************************/
+
+#define GET_HHIVE(CmHive)   (&((CmHive)->Hive))
+
+#if !defined(CMLIB_HOST) && !defined(_BLDR_)
+extern PCMHIVE CmiVolatileHive;
```

Comment:
Since that variable is used only once only there, it's cleaner to put it there, instead of polluting the global cmlib header with it.